src/components/PeoplePageContent.tsx

```diff
@@ -0,0 +1,32 @@
+import { usePeople } from '../providers/PeopleProvider';
+import { ErrorMessage } from './ErrorMessage';
+import { Loader } from './Loader';
+import { PeopleList } from './PeopleList';
+
+export const PeoplePageContent = () => {
+  const {
+    people, isError, isLoading,
+  } = usePeople();
+
+  if (isLoading) {
+    return <Loader />;
+  }
+
+  if (isError) {
+    return <ErrorMessage />;
+  }
+
+  if (people) {
+    if (people.length) {
+      return <PeopleList />;
+    }
+
+    return (
+      <p data-cy="noPeopleMessage">
+        There are no people on the server
+      </p>
+    );
+  }
+
+  return null;
```

Comment:
I think better will be to display <ErrorMessage> there than nothing and let user know